CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 ### 🛑 Breaking changes 🛑
 
 - `jmxreceiver`: Remove properties & groovyscript parameters from JMX Receiver. Add ResourceAttributes & LogLevel parameter to supply some of the removed functionality with reduced attack surface (#9685)
-
+- `awsxrayreceiver`: Removed deprecated `net.Error.Temporary()` reference from code.
```

Comment:
Is this really a breaking change? I think we can use the `Skip Changelog` label in this PR

Reply:
Not really - it is completely hidden , I am happy to remove Changelog

Reply:
@dmitryax removed changelog